crane-cloud/frontend
# Add proposed ux changes

# Description
Add proposed UX changes

## Type of change


## Trello Ticket ID
https://trello.com/c/HL6q9Vy7

## How Can This Been Tested?
Log into the platform taking a keen note of all the components on the pages to notice all the new changes

## Checklist:


# Screenshots
![Screenshot from 2022-10-12 16-59-08](https://user-images.githubusercontent.com/32802973/195373719-97932fe4-a0e5-4333-8017-3da104e7f812.png)
![Screenshot from 2022-10-12 16-51-14](https://user-images.githubusercontent.com/32802973/195373733-8dc338c7-1760-4de0-bbd3-c6e81580ef90.png)
